Resources/Textures/Objects/Consumable/Drinks/bloodglass.rsi/meta.json

```diff
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "Drawn by Ubaser", "states": [{"name": "icon"}]}
```

Comment:
formatting :trollface:

Reply:
just checked, its very standard for drink formatting? Its the same so im going to set it to resolved.